styles/styles.css

```
@@ -0,0 +1,539 @@
+Grijs #7A7A7A (voor de service-tegel tekst);
+/////////////////////////////////////////////////////////*/
+
+/*---------------------------------1. global styles----------------------------------------*/
+
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+:root {
+    --purple: #5722CD;
+    --lightpurple: #F2F0FF;
+    --blue: #4845E4;
+}
+
+/*---------------------------------2. typography----------------------------------------*/
+
+body {
+    font-family: 'Roboto', sans-serif;
+    color: #0C0C0C;
+}
+
+h1, h2, h3, h4 {
+    font-family: 'Merriweather', serif;
+}
+
+a:link {
+    text-decoration: none;
+    color: white;
+}
+
+/*---------------------------------3. areas----------------------------------------*/
+
+/*---------------------------------3.1 header----------------------------------------*/
+header {
+    background-color: var(--purple);
+    background-image: url("/assets/noise-texture.png"), linear-gradient(to right, var(--blue), hsl(259, 72%, 47%));
+    color: #F3F6F9;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+}
+
+nav {
+    width: 100vw;
+    max-width: 100%;
+}
+
+nav ul {
+    display: flex;
+    list-style: none;
+    justify-content: flex-end;
+}
+
+nav li {
+    margin: 20px 30px 20px 30px;
+}
+
+.intro {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 60px 0 0 0;
+    max-width: 700px;
+    margin: 0 40px;
+    flex: 0 1 auto;
+}
+
+.intro p {
+    margin: 20px 0 0 0;
+}
+
+.intro button {
+    font-size: 12px;
+    width: 200px;
+}
+
+.intro::after {
+    content: "";
+    background: url("/assets/squiggle_line_stripe.svg");
+    position: relative;
+    display: block;
+    height: 110px;
+    width: 70px;
+    top: 60px;
+    z-index: 1;
+}
+
+.top-triangle {
+    width: 100%;
+    height: 70px;
+    background: url("/assets/noise-texture.png"), linear-gradient(to right, var(--blue), hsl(259, 72%, 47%));
+    clip-path: polygon(0% 0%, 100% 0%, 100% 100%);
+}
+
+/*linear-gradient(to right, hsl(233, 45%, 52%), hsl(259, 72%, 47%));*/
+
+/*bij 100vw word de driehoek te groot. Bij de navigatiebalk kan ik het corrigeren
+met max-width: 100% dat lukt niet bij de driehoek.
+tussen de chrome en de firefox browser zit ook verschil??!*/
+
+/*---------------------------------3.2 main----------------------------------------*/
+
+.hello {
+    text-align: center;
+    color: var(--purple);
+    display: flex;
+    justify-content: center;
+    margin: 0 0 50px 0;
+}
+
+.hello article {
+    position: relative;
+}
+
+.hello article::before {
+    content: "hello";
+    color: var(--lightpurple);
+    font-size: 30vw;
+    font-weight: bold;
+    position: absolute;
+    left: -80px;
+    top: 0;
+    z-index: -1;
+
+}
+
+.hello .inner-container {
+    margin: 40px;
+    flex: 0 1 700px;
+}
+
+.hello article {
+    padding: 100px 0 100px 0;
```

Dit kun je ook korter schrijven:
```suggestion
    padding: 100px 0;
```